src/widget/wcuemenupopup.cpp

```diff
@@ -40,6 +54,52 @@ WCueMenuPopup::WCueMenuPopup(UserSettingsPointer pConfig, QWidget* parent)
     m_pDeleteCue->setObjectName("CueDeleteButton");
     connect(m_pDeleteCue, &QPushButton::clicked, this, &WCueMenuPopup::slotDeleteCue);
 
+    m_pStandardCue = new QPushButton("", this);
+    m_pStandardCue->setToolTip(
+            tr("Toggle this cue type to normal cue if not."));
+    m_pStandardCue->setObjectName("CueStandardButton");
+    m_pStandardCue->setCheckable(true);
+    connect(m_pStandardCue, &QPushButton::clicked, this, &WCueMenuPopup::slotStandardCue);
+
+    m_pSavedLoopCue = new CueTypePushButton(this);
+    m_pSavedLoopCue->setToolTip(
+            tr("Toggle this cue type to saved loop, using "
+               "the current beatloop size or the current play position") +
+            "\n\n" +
+            tr("Left-click: Toggle between normal cue and saved loop, using "
+               "the current beatloop size as the loop size if not previous size was known") +
+            "\n" +
+            tr("Right-click: Set the current play position as the loop end and "
+               "make the cue a saved loop if not"));
+    m_pSavedLoopCue->setObjectName("CueSavedLoopButton");
+    m_pSavedLoopCue->setCheckable(true);
+    connect(m_pSavedLoopCue, &CueTypePushButton::clicked, this, &WCueMenuPopup::slotSavedLoopCue);
+    connect(m_pSavedLoopCue,
+            &CueTypePushButton::rightClicked,
+            this,
+            &WCueMenuPopup::slotAdjustSavedLoopCue);
+
+    m_pSavedJumpCue = new CueTypePushButton(this);
+    m_pSavedJumpCue->setToolTip(
+            tr("Toggle this cue type between normal cue and saved jump, to "
+               "the current play position or the current beatjump size ") +
+            "\n\n" +
+            tr("Left-click: Toggle this cue type to saved loop, using "
+               "the current play position if not previous size was known. If "
+               "the play position is the cue position, uses the current "
+               "beatjump size") +
```

Comment:
(after reabsing) We need to add more line breaks here, the tooltip is currently very wide which makes reading a bit strenuous.

Reply:
also the current text doesn't mathc the Jump type.